tests/e2e/framework/kubernetes.go

```diff
@@ -696,6 +696,7 @@ func (k *KubeInfo) deployIstio() error {
 	}
 
 	if err := util.CheckDeployments(k.Namespace, maxDeploymentRolloutTime, k.KubeConfig); err != nil {
+		log.Errorf("CheckDeployments() failed a")
```

Comment:
The error message is weird, looks incomplete

Reply:
ya these two debug remanants need removal.

Reply:
done.